cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -107,6 +109,8 @@ runNode
   -> IO ()
 runNode cmdPc = do
 
+    Crypto.sodiumInit
+
```

Comment:
We should add a TODO and connect it to a ticket to change the crypto libs so we don't need to do this in applications. It should be done automagically by the lower level crypto lib.